src/shed/__init__.py

```diff
@@ -80,7 +80,8 @@ def shed(
     except Exception as err:
         msg = f"Could not parse {_location}\n    {type(err).__qualname__}: {err}"
         for pattern, blocktype in _SUGGESTIONS:
-            if re.search(pattern, source_code, flags=re.MULTILINE):
+            # TODO: write test for the below??
+            if re.search(pattern, source_code, flags=re.MULTILINE):  # pragma: no cover
```

Comment:
Ooh, it's not blocking for this PR but we should indeed write some tests for this: the idea is that if you have a console session or traceback in an rst `python` block, we can instead suggest using the correct `pycon` or `python-traceback` block type.  Follow-up PR?